src/support/SupportResourceListItem.ts

```diff
@@ -0,0 +1,63 @@
+import { Clickable } from '@terra-ui-packages/components';
+import * as qs from 'qs';
+import { useState } from 'react';
+import { div, h, span } from 'react-hyperscript-helpers';
+import colors from 'src/libs/colors';
+import * as Nav from 'src/libs/nav';
+import * as Style from 'src/libs/style';
+import { SupportResourceType } from 'src/support/SupportResourceType';
+
+const listItemStyle = (selected, hovered) => {
```

Comment:
```suggestion
const listItemStyle = (selected: boolean, hovered: boolean): CSSProperties => {
```

(`CSSProperties` is imported from react)